xarray/core/dataset.py

```diff
@@ -2720,6 +2848,32 @@ def tail(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Example
+        -------
+
+        # Sample dataset
+        >>> data = xr.DataArray([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dims=("x", "y"))
+        >>> dataset = xr.Dataset({"data": data})
+
+        # Print the original dataset
+        >>> dataset
+        <xarray.Dataset>
+        Dimensions:  (x: 3, y: 3)
+        Dimensions without coordinates: x, y
+        Data variables:
+            data     (x, y) int64 1 2 3 4 5 6 7 8 9
+
+        # Get the last 2 elements using tail()
+        >>> tail_dataset = dataset.tail(2)
```

Comment:
Good. I think just finishing with `dataset.tail(2)` without reassigning to a new python variable is fine though.